lib/wickedbase.pm

```diff
@@ -554,9 +580,8 @@ sub wait_for_dhcpd {
 
 sub ifbind {
     my ($self, $action, $interface) = @_;
-    my $cmd = "./ifbind.sh $action $interface";
-    record_info('bind', $cmd);
-    assert_script_run($cmd);
+    my $cmd = 'bash ' . WICKED_DATA_DIR . "/wicked/ifbind.sh $action $interface";
+    record_info('bind', $cmd . "\n" . script_output($cmd));
```

Comment:
```suggestion
    record_info('ifbind', $cmd . "\n" . script_output($cmd));
```
otherwise it could be confusing when you call "unbind" but title is bind